src/main/java/org/concordion/internal/listener/ThrowableRenderer.java

```diff
@@ -106,13 +104,6 @@ private void recursivelyAppendStackTrace(Throwable t, Element stackTrace) {
             stackTrace.appendChild(stackTraceElement(traceElement));
         }
 
-        if (t instanceof OgnlException) {
-            Throwable reason = ((OgnlException) t).getReason();
-            if (reason != null) {
-                recursivelyAppendStackTrace(reason, stackTrace);
-            }
-        }
-        

```

Comment:
Removing this will make the stack traces harder to understand. How else can this be structured so that we retain this code for the Java version?